plugins/janus_videoroom.c

```diff
@@ -2830,26 +2697,7 @@ struct janus_plugin_result *janus_videoroom_handle_message(janus_plugin_session
 		}
 		/* Compute a list of the supported codecs for the summary */
 		char audio_codecs[100], video_codecs[100];
-		memset(audio_codecs, 0, sizeof(audio_codecs));
-		g_snprintf(audio_codecs, sizeof(audio_codecs), "%s", janus_videoroom_audiocodec_name(videoroom->acodec[0]));
-		if(videoroom->acodec[1] != JANUS_VIDEOROOM_NOAUDIO) {
-			g_strlcat(audio_codecs, "|", sizeof(audio_codecs));
-			g_strlcat(audio_codecs, janus_videoroom_audiocodec_name(videoroom->acodec[1]), sizeof(audio_codecs));
-		}
-		if(videoroom->acodec[2] != JANUS_VIDEOROOM_NOAUDIO) {
-			g_strlcat(audio_codecs, "|", sizeof(audio_codecs));
-			g_strlcat(audio_codecs, janus_videoroom_audiocodec_name(videoroom->acodec[2]), sizeof(audio_codecs));
-		}
-		memset(video_codecs, 0, sizeof(video_codecs));
-		g_snprintf(video_codecs, sizeof(video_codecs), "%s", janus_videoroom_videocodec_name(videoroom->vcodec[0]));
-		if(videoroom->vcodec[1] != JANUS_VIDEOROOM_NOVIDEO) {
-			g_strlcat(video_codecs, "|", sizeof(video_codecs));
-			g_strlcat(video_codecs, janus_videoroom_videocodec_name(videoroom->vcodec[1]), sizeof(video_codecs));
-		}
-		if(videoroom->vcodec[2] != JANUS_VIDEOROOM_NOVIDEO) {
-			g_strlcat(video_codecs, "|", sizeof(video_codecs));
-			g_strlcat(video_codecs, janus_videoroom_videocodec_name(videoroom->vcodec[2]), sizeof(video_codecs));
-		}
+		janus_videoroom_codecstr(videoroom, audio_codecs, video_codecs, 100, "|");
```

Comment:
I'd put `sizeof(video_codecs)` or `sizeof(audio_codecs)` there instead of 100, just in case in the future we change the size of those arrays and forget to update it in this call too.